src/components/generic/Header.tsx

```diff
@@ -336,6 +336,26 @@ class Header extends React.Component<Props, State> {
     }
   }
 
+  /**
+   * Method for comparing menuorder from CustomPage. Used to sort pages
+   * 
+   */
```

Comment:
Add parameters and return value to doc and also specify method return value type. For example `private func = (): string => { return "hello" }` has return type of string.

Reply:
Made changes